app/boards/shields/romac_plus/romac_plus.keymap

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2020 Pete Johanson, Richard Jones
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+#include <behaviors.dtsi>
+#include <dt-bindings/zmk/keys.h>
+
+/ {
+    keymap {
+        compatible = "zmk,keymap";
+        
+        default_layer {
+// -------------------
+// | 7      |  8  |  9  |
+// | 4      |  5  |  6  |
+// | 1      |  2  |  3  |
+// | MO(1)  |  0  |  .  |
+// ----------------------
+            bindings = <
+    &kp NUM_7 &kp NUM_8 &kp NUM_9 
+    &kp NUM_4 &kp NUM_5 &kp NUM_6 
+    &kp NUM_1 &kp NUM_2 &kp NUM_3 
+    &cp M_PLAY &kp NUM_0 &kp DOT 
+            >;
+
+            sensor-bindings = <&inc_dec_cp M_NEXT M_PREV>;
+        };
+
+        // nav_layer {
+// -----------------------
+// | _ |  HOME  |  PGUP  |
```

Comment:
We're trying to include the `&bt` bindings, at least the "clear" one, in stock keymaps now where we can. See https://zmkfirmware.dev/docs/behavior/bluetooth/

Reply:
Alrighty. Added it to a lower layer to prevent accidental clearing in [9fffebd](https://github.com/zmkfirmware/zmk/pull/198/commits/9fffebd5dac13d0a57f6f6aaff0654a2bd0986c1) 

It was a bit finnicky if you already had a device paired with the previous firmware (before these changes). I could not simply press the reset button once. I had to use BT_CLR in order to repair the device. With this expectation change I guess some people might have issues.